wordpress-mobile/gutenberg-mobile
# Create script to automate release process steps.

This PR adds a script to the repo to automate all the steps in the creation of the step process.

The idea is that on the cut release day you go to the `develop` branch and run this command:

`./bin/release_automation.sh`

It assumes the following:
 
 - npm package `json` is installed. This is done by running `yarn install` before
 - the [GitHub CLI is pre-installed](https://cli.github.com/). This needs to be done manually before using the script.

To test:
 - Check the content of the script
 - If you want to give it a go run the following: `./bin/release_automation.sh`
 - Remember to delete the PR just created and the corresponding branch.

PR submission checklist:
